samples/Sentry.Samples.Console.Basic/Program.cs

```diff
@@ -33,8 +34,37 @@
 
     // This option tells Sentry to capture 100% of traces. You still need to start transactions and spans.
     options.TracesSampleRate = 1.0;
+
+    options.EnableLogs = true;
+    options.SetBeforeSendLog(static (SentryLog log) =>
+    {
+        if (log.TryGetAttribute("plan.type", out string? attribute) && attribute == "enterprise")
+        {
+            return null;
+        }
+
+        return log.Level is SentryLogLevel.Error ? log : null;
+    });
 });
 
+var configureLog = static (SentryLog log) =>
+{
+    log.SetAttribute("string-attribute", "value");
+    log.SetAttribute("boolean-attribute", true);
+    log.SetAttribute("integer-attribute", long.MaxValue);
+    log.SetAttribute("double-attribute", double.MaxValue);
+};
+
+SentrySdk.Logger.LogTrace("Hello, World!", null, configureLog);
+SentrySdk.Logger.LogDebug("Hello, .NET!", null, configureLog);
+SentrySdk.Logger.LogInfo("Information", null, configureLog);
+SentrySdk.Logger.LogWarning("Warning with one {0}", ["parameter"], configureLog);
+SentrySdk.Logger.LogError("Error with {0} {1}", [2, "parameters"], configureLog);
+SentrySdk.Logger.LogFatal("Fatal {0} and {1}", [true, false], configureLog);
+
+await Task.Delay(TimeSpan.FromSeconds(5));
```

Comment:
Do we need this wait?
If u want to flush, there's:

```suggestion
await SentrySdk.FlushAsync(TimeSpan.FromSeconds(5));
```

Reply:
I reworked the sample a bit to be less verbose.